documentation/adr/0002-wiki-platform.md

```diff
@@ -0,0 +1,294 @@
+# Communications Tooling: Wiki Platform
```

Comment:
The more I think about it, the more I think we move this into a separate PR. Or just merge it incomplete and open a new PR to actually make the decision.

Reply:
Agreed, I wasn't planning to really make any more edits to this ADR in this branch. Do you think it would be better to leave as is, or just copy the data into a different branch and delete this file?

Reply:
Either is fine for me! It seems simpler to leave as is and then open a new branch to update the existing file, but either works.

Reply:
This file was moved into #30